tests/test_models/test_autoarima_model.py

```diff
@@ -114,11 +102,13 @@ def test_prediction_interval_run_infuture_autoarima(example_tsds):
         assert (segment_slice["target_0.975"] - segment_slice["target_0.025"] >= 0).all()
 
 
-def test_forecast_raise_error_if_not_fitted_autoarima(example_tsds):
-    """Test that AutoARIMA raise error when calling forecast without being fit."""
+@pytest.mark.parametrize("method_name", ["forecast", "predict"])
+def test_prediction_raise_error_if_not_fitted_autoarima(example_tsds, method_name):
```

Comment:
Do we actually need this test?

Reply:
I don't think that getting rid of tests without a reason is a good idea.

Reply:
Probably it is better to check that we fail in this strange case then getting some strange results.